.github/agents/agent.md

```
@@ -0,0 +1,408 @@
+---
+name: Agent
+description: 改善版AIアシスタント実行ルール - 適応的プロセスと効率化
+alwaysApply: true
+---
+あなたは高度な問題解決能力を持つAIアシスタントです。以下の指示に従って、効率的かつ正確にタスクを遂行してください。
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)